pkg/network/tracer/tracer.go

```diff
@@ -252,13 +252,17 @@ func newConntracker(cfg *config.Config) (netlink.Conntracker, error) {
 			log.Warnf("failed to load conntrack kernel module, though it may already be loaded: %s", err)
 		}
 	}
-
-	if c, err = NewEBPFConntracker(cfg); err == nil {
-		return c, nil
+	if cfg.EnableEbpfConntracker {
+		if c, err = NewEBPFConntracker(cfg); err == nil {
+			return c, nil
+		}
+		log.Warnf("error initializing ebpf conntracker: %s", err)
+	} else {
+		log.Info("ebpf conntracker disabled via config")
 	}
 
 	if cfg.AllowNetlinkConntrackerFallback {
```

Comment:
This is not related to this PR, but there looks like a gap here. What if the netlink fallback is disabled? We seem to not handling that case.

Reply:
what is the value of disabling the fallback? should we even support that anymore? even if netlink is broken on a host you could set `IgnoreConntrackInitFailure` to true so sysprobe can start

Reply:
I think the case for setting `AllowNetlinkConntrackerFallback` to `false` would be that we don't want to use the netlink conntracker, only the eBPF conntracker. But I am not sure that is really important to anyone.

I see two options:
1. we get rid of this flag (`AllowNetlinkConntrackerFallback`), although I am not sure if that breaks anyone
2. report a sane log/end up in a sane state if somehow a misconfiguration has disabled both conntrackers

Reply:
I think I am in favor of option 1 because:
1. with my PR from yesterday we don't really have a path to phasing out netlink conntracker for the foreseeable future
2. When we're ready, we can always bring back this config... right now it just clutters the code
3. I would be very surprised if any customers are using this config? I could be wrong